bitmap-goldteam/model/bitmap-constructor.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = exports = {};
+
+exports.readContent = function(callback) {
+  fs.readFile(`${__dirname}/../img/palette-bitmap.bmp`, function(err, data) {
```

Comment:
@GLSea1979 @0blu3 @noahgribbin the function that you use to read a bitmap and subsequently add data as part of the constructor should be extracted out of this `readContent` method - it's best to create a helper file (ex: `bitmap-file-reader`) that encapsulates your `readFile` call and allows you to dynamically pass in a filename for reuse in other modules and for proper testing